test/shared_utilities/file_reader.jl

```diff
@@ -27,23 +27,23 @@ isdir(regrid_dir_temporal) ? nothing : mkpath(regrid_dir_temporal)
 FT = Float32
 @testset "test interpol, FT = $FT" begin
```

Comment:
I only replaced `t` with `x` in this test bc it kept coming up when I searched for `t = FT...`

Reply:
:D